data/apps/scriptable.json

```diff
@@ -0,0 +1,56 @@
+{
+  "info": {
+    "identifier": "scriptable",
+    "name": "Scriptable",
+    "appstoreUrl": "https://itunes.apple.com/au/app/scriptable/id1405459188?mt=8"
```

Comment:
Is there no documentation link? Where do you have the docs from?

Reply:
The documentation is documented within the app so I couldn't link to it. I searched Google for other documentation, but didn't find any. Here's a copy/paste of the documentation from within the app:

### URLScheme
**Manages URL schemes for Scriptable.**

Use URL schemes to launch the app and perform an action, such as running a script.
The app conforms to the scriptable:// URL scheme. The following documents actions that can be performed using the URL scheme.

**Adding a script**
To add a new script, you should use the following URL scheme:

`scriptable:///add`

**Opening a script**
To open an existing script, you should use the following URL scheme:

`scriptable:///open?scriptName=Example`

The scriptName query parameter is the name of the script to open. scriptName must be URL encoded. You may optionally add the query parameter openSettings with a value of true to automatically open the script settings.

**Running a script**
To run an existing script, you should use the following URL scheme:

`scriptable:///run?scriptName=Example`

The scriptName query parameter is the name of the script to run. scriptName must be URL encoded.

**Screenshot**
![image](https://user-images.githubusercontent.com/546216/46598184-942d6980-cb26-11e8-834e-0197f4302e97.png)

Reply:
Cool thanks for the explanation! Once we have the docs in here, I'll try and reach out to the app maker and ask if they want to provide the docs somewhere outside the app too.

If you could just fix the `///` to `//` then I can merge and deploy this